test/cmake/file-compress.cmake

```diff
@@ -0,0 +1,271 @@
+# compress-and-verify.cmake -- Runs a test against an input file to make sure that the specified
```

Comment:
What is different between this file and `compress-and-verify.cmake`?

Reply:
This file works on files, not streams.